src/Entity/Main/Booking.php

```diff
@@ -22,6 +22,8 @@ class Booking
 
     private \DateTime $endTime;
 
+    private string $webformSubmission;
```

Comment:
Can we split this into the required fields instead of a blob of everything?
userMail
userName
metaData

Then we could avoid the json encoding.

Reply:
Changed